aat/src/aat/resources/features/F-083/S-359.td.json

```diff
@@ -0,0 +1,48 @@
+{
+  "_guid_": "S-359",
+  "_extends_": "F-083_Test_Base_Data",
+  "title": "must return the case type for an appropriate request",
+  "specs": [
+    "an active profile in CCD",
+    "contains a valid case type id",
+    "has the 200 OK code"
+  ],
+  "users": {
+    "invokingUser": {
+      "username": "befta.caseworker.3@gmail.com",
+      "password": "[[$CCD_BEFTA_CASEWORKER_3_PWD]]"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 200,
+    "headers" : {
+      "Cache-Control" : "[[ANYTHING_PRESENT]]",
+      "Content-Encoding" : "[[ANYTHING_PRESENT]]",
+      "Content-Length" : "[[ANYTHING_PRESENT]]",
+      "Content-Type" : "[[ANYTHING_PRESENT]]",
+      "Date" : "[[ANYTHING_PRESENT]]",
+      "Expires" : "[[ANYTHING_PRESENT]]",
+      "Pragma" : "no-cache",
+      "Vary" : "accept-encoding",
+      "X-Content-Type-Options" : "nosniff",
+      "X-Frame-Options" : "DENY",
+      "X-Xss-Protection" : "1; mode=block"
+    },
+    "body": {
+      "case_type_id" : "BEFTA_CASETYPE_3_4",
+      "fields" : [ {
+        "case_field_id" : "TextField",
+        "case_field_element_path" : null,
+        "label" : "Search `Text` field",
+        "order" : 1,
+        "role" : null
+      }, {
+        "case_field_id" : "[LAST_STATE_MODIFIED_DATE]",
```

Comment:
Nice one 👍 . Perhaps add a comment in the feature file that it can be re-use / extend to verify other fields in the search functionality.